cluster/validate-cluster.sh

```diff
@@ -32,15 +32,24 @@ attempt=0
 while true; do
   # The "kubectl get nodes" output is three columns like this:
   #
-  #     NAME                     LABELS    STATUS
-  #     kubernetes-minion-03nb   <none>    Ready
+  #     NAME                     LABELS    STATUS   AGE
+  #     kubernetes-minion-03nb   <none>    Ready    10m
   #
   # Echo the output, strip the first line, then gather 2 counts:
   #  - Total number of nodes.
   #  - Number of "ready" nodes.
   "${KUBE_ROOT}/cluster/kubectl.sh" get nodes > "${MINIONS_FILE}" || true
   found=$(cat "${MINIONS_FILE}" | sed '1d' | grep -c .) || true
-  ready=$(cat "${MINIONS_FILE}" | sed '1d' | awk '{print $NF}' | grep -c '^Ready') || true
+
```

Comment:
Good point, this should really have been using `-o template` in the first place (and would then have been unaffected by your change). The following command should be useful as a starting point:
```
kubectl get nodes -o template --template='{{range .items}}{{$cond := index .status.conditions 0}}{{$cond.type}},{{$cond.status}}:{{end}}' --api-version=v1
```